src/main/java/entity/User.java

```diff
@@ -76,6 +76,10 @@ public User(
     this(-1, username, null, emailID, mobileNo, base64Image);
   }
 
+  public User(String username, String email, String picture) {
```

Comment:
With so many constructors here, why not use static factory methods or better yet, Builder pattern. (This can be a separate PR too since this is independent of this change.)

Reply:
Will create a separate PR.👍